docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md

```diff
@@ -22,7 +22,15 @@ A universe's **Metrics** page displays graphs representing information on operat
 
 You access metrics by navigating to **Universes > Universe-Name > Metrics**, as shown in the following illustration:
 
-![Metrics Page](/images/yp/metrics.png)<br>
+![Metrics Page](/images/yp/metrics_main.png)<br>
+
+You can chose to view metrics based on different criteria:
+
+- If your universe has more than one region, cluster, zone, or node, the **Metrics** page allows you to view metrics for a specific region, cluster, zone, or node.
+- You can view the overall node metrics.
+- You can view metrics for up to five outlier nodes. For example, to display charts for the top three nodes with the highest CPU usage, you would select **Outlier Nodes**, then **Top**, and then **3 nodes**. The chart also includes the average for all nodes.
```

Comment:
Thanks, @rajmaddy89. I implemented some of your suggestions.
WRT additional screenshots, we try to avoid overloading doc pages with images. In this particular case, the UI seems pretty intuitive, it's described, and it is very likely that the user is already looking at the UI while reading the doc.